dts/bindings/ethernet/nxp,mcux-ethernet.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Add license & copyright header info, see dts/bindings/gpio/gpio-leds.yaml for an example